build.gradle

```diff
@@ -125,6 +128,20 @@ if (hasProperty("buildScan")) {
 	}
 }
 
+// CRL schema extraction
+configurations {
+	cloudResourceSchema { transitive = false }
+}
+dependencies {
+	cloudResourceSchema group: 'bio.terra.cloud-resource-lib', name: 'cloud-resource-schema'
+}
+task unzipCloudResourceSchema(type: Copy) {
+	from zipTree(configurations.cloudResourceSchema.singleFile).matching {
+		include 'cloud_resources_uid.yaml'
+	}
+	into "${resourceDir}/api/common"
```

Comment:
Why do we need to unpack this into the source tree?
Is an alternative to copy our resources into the build tree?

Reply:
I unpacked it into the source tree so that our swagger-ui (also in the source dir) can parse references to it. otherwise, those schemas won't show up in swagger-ui, which breaks example values.

Reply:
We can leave it for now. I'll make a ticket.
Another approach might be to move all of the parts into a build subdirectory and do the codegen from there.
I'll make a ticket.